alumnium/__init__.py

```diff
@@ -1,11 +1,17 @@
 import logging
-import os
-import sys
+from os import getenv
+from sys import stdout
 
-if os.getenv("ALUMNIUM_DEBUG", "0") == "1":
-    logger = logging.getLogger(__name__)
-    logger.setLevel(level=logging.DEBUG)
-    logger.addHandler(logging.StreamHandler(sys.stdout))
+
+logger = logging.getLogger(__name__)
+
+level = getenv("ALUMNIUM_LOG_LEVEL", None)
+if getenv("ALUMNIUM_DEBUG", "0") == "1":
```

Comment:
Is it kept for backward compatibility?

Reply:
Not really, I can remove it if you think it's unuseful.

Reply:
I think it might be more confusing than helpful. 

I imagine the situation when `ALUMNIUM_DEBUG=1` and `ALUMNIUM_LOG_LEVEL=info`.  If I look at them in the `.env` file, I can forget which one gets a priority